### ISSUE TITLE
Address accessibility issues

### ISSUE THREAD

**Issue by richnoah** [w3c/aria-at-app#331]
There were several issues noted in an accessibility audit by PAC.

A summary list of the accessibility items includes:

- Missing/Ultimately Non-Descriptive Page Title
- Problematic Article Mark-Up
- Non-Descriptive Alternative Text for Image
- Potentially Incorrect Positioning of Image
- Missing Breadcrumbs Heading and List Accessible Name
- Inconsistent Page Title
- Unclear Main Heading
- Confusing Presence of Empty Headings and Table
- Inconsistent Positioning of Test Plan Actions
- Missing Programmatic Association Between Tester Usernames and Text
- Use of All Caps Text
- Potentially Unclear Link Purpose
- Presence of Single-Item List
- Missing Screen Reader Feedback when Content is Loaded
- Non-Descriptive Page Title
- Problematic Surfacing of Test Statuses
- Mismatched Accessible Names and Visible Labels for Tests
- Missing Heading and List Mark-Up for Button Group
- Non-Descriptive Footer Content
- Problematic Heading Mark-Up
- Incorrect Application of tabindex on Heading
- Missing Skip Link
- Missing Programmatic Association Between "Sign out" Link and Text
- Missing Heading and List Mark-Up for Navigation Items
- Problematic Positioning of " Signed in as …" Text
- Inconsistent Rendering of Breadcrumbs/Presence of Single-Item List
